oluseyianani/stackoverflow-lite
# #159844187 Fix server error on POST and  GET request

#### What does this PR do?
A fix to the server error on Post or get request
#### Description of Task to be completed?
When users make a request to the server, there should not be a server error
#### How should this be manually tested?
After cloning the repo, cd into it and RUN `npm run production`
- Use POSTMAN to make a POST request to `localhost:8080/api/v1/:id/answers`
/:id: Should be a number
_KEY_: answers _VALUE_: <any string of your choice>
#### Any background context you want to provide?
There must be a question of that particular id available to make a POST request to it.
#### What are the relevant pivotal tracker stories?
#159844187
#### Screenshots (if appropriate)
![err fixed](https://user-images.githubusercontent.com/25327188/44422331-bff68f00-a57a-11e8-9fb5-775396ea49e8.png)
#### Questions:
N/A
